lang/en.json

```diff
@@ -1,25 +1,25 @@
 {
     "VisitorGenerator": {
-        "PluginDescription": "Create fake visits and actions in Matomo. This can be used by Matomo users or developers as an easy way to generate fake data to populate Matomo reports.",
+        "PluginDescription": "Create fake visits and actions in Matomo. Generate fake data to populate Matomo reports.",
         "VisitorGenerator": "Visitor Generator",
-        "Warning": "You are about to generate fake visits which will be recorded in your Matomo database.",
+        "Warning": "You are about to generate fake visits for your Matomo database.",
         "NotReversible": "It will %1$s not%2$s be possible to easily delete these visits from the Matomo logs.",
-        "AreYouSure": "Are you sure you want to generate fake visits?",
-        "ChoiceYes": "Yes, I am sure!",
+        "AreYouSure": "Generate fake visits?",
+        "ChoiceYes": "Yes",
         "Submit": "Submit",
-        "SelectWebsite": "Select Website",
+        "SelectWebsite": "Select website",
         "NbRequestsPerSec": "Requests per second",
         "DaysToCompute": "Days to compute",
         "GenerateFakeActions": "This will generate at least %d fake actions on this site for each day.",
-        "OverwriteLogFiles": "You can overwrite the log files that are used to generate fake visits (change or add files in %s). There are log files of requests to \"matomo.php\" in the format \"Apache combined log\". Plugin developers can provide their own log files by placing '*.log' files within a 'data' directory of their plugin.",
-        "PleaseBePatient": "NOTE: It might take a few minutes to generate visits and actions, please be patient...",
-        "LogImporterNote": "There is also a faster tool that will import large test data in Matomo, see the %1$sREADME%2$s.",
-        "AutomaticReprocess": "The reports will be reprocessed the next time you visit the Matomo reports, it might take a few minutes.",
+        "OverwriteLogFiles": "You can overwrite the log files used to generate fake visits (change or add files in %s). There are log files of requests to \"matomo.php\" in the format \"Apache combined log\". Plugin developers can provide their own log files by placing '*.log' files within a 'data' directory of their plugin.",
+        "PleaseBePatient": "Generating visits and actions…",
```

Comment:
```suggestion
        "PleaseBePatient": "Generating visits and actions, this may take a few minutes…",
```

Reply:
That might need a rewording due to the way this is currently used. It is not shown while visits are generated, but as a note, before that is done.

This is how the page using most of the translations here currently looks like:
![image](https://user-images.githubusercontent.com/1579355/188465032-dd3eebf0-4b37-4d04-a1e9-4aa0cab38caf.png)

@comradekingu maybe you can think of some further adjustments based on that.